README.md

```diff
@@ -0,0 +1,35 @@
+# Remix Compute for Netlify
+
+The Remix Compute for Netlify packages allows you to deploy your Remix apps to Netlify on the Edge.
```

Comment:
```suggestion
The Remix Compute for Netlify packages allow you to deploy your Remix apps to Netlify on the Edge.
```

Typo. Also I'm not sure about naming. Do we need "Compute"? I don't see any precedent for this naming convention. Wouldn't "Remix for Netlify" do the job?